requirements.in

```diff
@@ -6,9 +6,3 @@ requests==2.22.0
 google-cloud-bigquery==1.24.0
 google-cloud-core==1.3.0
 mozanalysis==2020.2.1
-
-coverage==5.0.3
-pytest==5.3.5
-pytest-black==0.3.8
```

Comment:
I ran into an issue when installing `black`. One of its dependencies requires `regex` which needs `gcc` to build. However, installing `gcc` in miniconda/conda is a bit of a pain. So I decided to split up the requirements files